test/network/setup/config.js

```diff
@@ -36,7 +36,7 @@ const SYNC_MODES = {
 
 const SYNC_MODE_DEFAULT_ARGS = {
 	RANDOM: {
-		probability: 0.5, // (0 - 1)
+		probability: 0, // (0 - 1)
```

Comment:
It's not RANDOM then :)

Reply:
Actually, I should remove that commit. It doesn't change anything. I misunderstood the parameter.